common/lib/mandrill_client/client.py

```diff
@@ -38,40 +46,66 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def get_email_to(self, receiver_emails_string):
+        """
+        parsing the comma separated email to a list of emails
+
+        Arguments:
+            receiver_emails_string: String contains a comma separated emails
+
+        Returns:
+            receiver_emails (List): A List of receiver email
+        """
+        email_list = receiver_emails_string.split(',')
+        receiver_emails = [{'email': email} for email in email_list]
+        return receiver_emails
+
+    def send_mail(self, template_name, receiver_emails_string, context, attachments=None, subject=None,
+                  reply_to_email=None, images=None):
```

Comment:
I hope we already have verified that `send_email` isn't being called with named parameters. If not please verify so that it won't break somewhere. If yes, Please ignore this comment

Reply:
yes we have verified it, we are not using named parameter any where in the project.